pkg/scheduler/ugm/queue_tracker.go

```diff
@@ -89,13 +89,16 @@ func (qt *QueueTracker) increaseTrackedResource(hierarchy []string, applicationI
 		}
 	}
 
+	if resources.IsZero(qt.resourceUsage) {
```

Comment:
We can just use `qt.resourceUsage == nil`. If `qt.resourceUsage` is empty or all resources is 0, it will pass `resources.IsZero` and we will reassign a similar value to it.

https://github.com/apache/yunikorn-core/blob/0fd743614b973650fc3fa24488be240e2e0b3121/pkg/common/resources/resources.go#L930-L942